src/app/__tests__/app.component.spec.ts

```diff
@@ -1,32 +1,44 @@
-import { TestBed, ComponentFixture } from '@angular/core/testing';
+import { TestBed, ComponentFixture, async } from '@angular/core/testing';
```

Comment:
how comes we need to import `async` from here? It's just part of TypeScript no?

Reply:
Without that `async` function the tests are failing, because the tests start to run before the test bed gets configured. If we just declare it as an async function nothing is waiting for the setup to be complete.